### ISSUE TITLE
internationalization: re-use of included standard i18n strings

### ISSUE THREAD

**Issue by tbba** [formvalidation/support#277]
Not sure I asked that before.
I make heavy use on the predefined language strings per language localization.

For individual validators I would like to avoid going through the hassle of setting up individual messages in every given language and have them translated - that need to be maintained if languages are added to the webiste. The information given in the strings delivered with your script just work fine in many cases, like "not empty, wrong format, wrong email format..."

(i.e. my special regex "email checker" validator sends back the error messages of the standard email checker validator.)

**Comment by nghuuphuoc** [comment]
Yes, you asked that in [i18n add-on](https://github.com/formvalidation/addon-i18n/issues/1). I will think of a way to reuse the language items in other language.
